js/ai/src/telemetry.ts

```diff
@@ -216,6 +216,18 @@ export function recordGenerateActionOutputLogs(
   const candidates = output.candidates.length;
   output.candidates.forEach((cand, candIdx) => {
     const parts = cand.message.content.length;
+    const candCounts = parts > 1 ? ` (${candIdx + 1} of ${parts})` : '';
+    logger.logStructured(`Output Candidate[${path}, ${model}]${candCounts}`, {
+      ...sharedMetadata,
+      candidateIndex: candIdx,
+      totalCandidates: candidates,
```

Comment:
Might be able to leave these two props out, because Genkit is moving away from multiple candidates.